docs/customization.md

```diff
@@ -36,7 +36,7 @@ organized with a subdirectory per variant.
 Environment files are named using the [canonical distribution
 name](#canonical-distribution-names) and the suffix `.env`.
 
-```
+```console
```

Comment:
are these meant to be bash? they include both bash commands and log.

Reply:
I thought console was for that case? I honestly don't know, but this stopped the linter from complaining.

Reply:
+1 `console` is an alias for `ShellSession`, which is designed to high-light interactive session in a Unix shell.

Reply:
Wonderful, thank you both!